src/content/index.js

```diff
@@ -161,6 +166,8 @@ function observeBody() {
         })
         addPlayerProfileDownloadDemo(mainContentElement)
         addPlayerProfileExtendedStats(mainContentElement)
+      } else if (pages.isPlayerProfile()) {
+        addPlayerProfileBadge(mainContentElement)
```

Comment:
I'm sorry, I didn't realize that it was actually correct when it was nested, so `addPlayerProfileBadge` is only called once when it's the player profile in general. Would you mind reverting this back to the previous state please.